src/transformer/events/mod_feedback/item_answered/textfield.php

```diff
@@ -47,7 +47,7 @@ function textfield(array $config, \stdClass $event, \stdClass $feedbackvalue, \s
         ],
         'timestamp' => utils\get_event_timestamp($event),
         'result' => [
-            'response' => $feedbackvalue->value,
+            'response' => ($feedbackvalue->value ? $feedbackvalue->value:''),
```

Comment:
This could be simplified a bit by using `?:`.

```suggestion
            'response' => ($feedbackvalue->value ?: ''),
```

Reply:
OK - made this change